sdcflows/workflows/b0.py

```diff
@@ -0,0 +1,91 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+
+
+"""
+
+from nipype.pipeline import engine as pe
+from nipype.interfaces import utility as niu, fsl, ants
+from nipype.workflows.dmri.fsl.utils import demean_image, cleanup_edge_pipeline
+from niworkflows.engine.workflows import LiterateWorkflow as Workflow
+from niworkflows.interfaces.bids import DerivativesDataSink
+from niworkflows.interfaces.images import IntraModalMerge
+from niworkflows.interfaces.masks import BETRPT
+from ..interfaces.fmap import FieldEnhance
+
+
+def init_prepare_magnitude_wf(omp_nthreads, name='magnitude_wf'):
+    """Merge, bias correct and skull-strip the magnitude image(s)."""
+    workflow = Workflow(name=name)
+    inputnode = pe.Node(
+        niu.IdentityInterface(fields=['magnitude', 'source_file']), name='inputnode')
+    outputnode = pe.Node(niu.IdentityInterface(fields=['fmap_ref', 'fmap_mask']),
+                         name='outputnode')
+
+    # Merge input magnitude images
+    magmrg = pe.Node(IntraModalMerge(), name='magmrg')
+
+    # de-gradient the fields ("bias/illumination artifact")
+    n4_correct = pe.Node(ants.N4BiasFieldCorrection(dimension=3, copy_header=True),
```

Comment:
It might be worth using [`niworkflows.interfaces.fixes.FixN4BiasFieldCorrection`](https://github.com/poldracklab/niworkflows/blob/d280796b07297a8a59b47fc5951cda06880b66ed/niworkflows/interfaces/fixes.py#L76) using `negative_values=True` to make sure negative values do not sneak into this step. Also, it would be interesting to have a smaller shrinking because voxels are generally larger than T1w images - we could add this in the next iteration and fix it of other existing workflows, if you prefer.

If you prefer not to take the suggestion this time, please open up a new issue requesting this.